pages/aws/common_issues.mdx

```diff
@@ -62,3 +62,35 @@ If you are refreshing a dynamic/static route or going to that route directly fro
 This can also happen in app router when a client navigates via NextJS `<Link>` component.
 
 The issue might be that your having a folder or file in your `public` directory with an overlapping between the name and your route. In this case, you should rename that to something else.
+
+
+#### Sentry does not work for api/server side + `cannot find module './chunks/3381.js'` error
```

Comment:
This does not affect only api, it will cause issue for everything server side related
```suggestion
#### Sentry does not work server side + `cannot find module './chunks/xxxx.js'` error
```